### DISCUSSION TITLE
How do I pass arbitrary Vite command line options when calling remix vite:dev?

### DISCUSSION THREAD

**Discussion by meglio** [remix-run/remix#8944] [Q&A]
Vite has quite a few command line options: https://vitejs.dev/guide/cli.html

The documentation does not explain how to pass them through when calling `remix vite:dev`.

How to?

**Comment by kiliman** [comment]
It doesn't look like you can currently. Remix passes options to the `dev` function, but it _destructures_ them, so they would be ignored even if you passed in additional options. The same thing happens for the `build` command.

Perhaps this could be a new feature request.

https://github.com/remix-run/remix/blob/dbdf223429138044139b4cd4e6d6dbe4a1e2449e/packages/remix-dev/vite/dev.ts#L21-L35
```ts
export async function dev(
  root: string,
  {
    clearScreen,
    config: configFile,
    cors,
    force,
    host,
    logLevel,
    mode,
    open,
    port,
    strictPort,
  }: ViteDevOptions
) {
```

**Reply by meglio** [reply]
Is there a way to promote this discussion into a feature request?

**Comment by roll** [comment]
For some subset(?) of the options a env var can be used, for example:

```
DEBUG=vite:hmr react-router dev
```
